src/commands/info.ts

```diff
@@ -1,16 +1,137 @@
-import type { FluorineClient } from '#classes';
-import type { Category } from '#types';
-import { getInfoComponents, getInfoEmbed } from '#util';
-import { type ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
-
-export async function run(client: FluorineClient, interaction: ChatInputCommandInteraction) {
-    interaction.reply({
-        embeds: [await getInfoEmbed(client, interaction, 'info')],
-        components: [getInfoComponents(client, interaction, 'info')]
-    });
+import { readdir } from 'node:fs/promises';
+import { join } from 'node:path';
+import process from 'node:process';
+import { Embed, type FluorineClient } from '#classes';
+import type { Category, ComponentData } from '#types';
+import { getDirname } from '#util';
+import {
+    type ChatInputCommandInteraction,
+    type Interaction,
+    type ButtonInteraction,
+    SlashCommandBuilder,
+    ActionRowBuilder,
+    ButtonBuilder,
+    ButtonStyle
+} from 'discord.js';
+
+async function getEmbed(client: FluorineClient, interaction: Interaction, page: string) {
```

Comment:
can we get rid of the getembed and getcomponents functions (have the embed/component in oninteraction)?